src/main/java/net/datafaker/service/RandomService.java

```diff
@@ -69,6 +67,19 @@ public double nextDouble(double min, double max) {
         return min + (nextDouble() * (max - min));
     }
 
+    public double nextGaussian() {
```

Comment:
What's the use case here, and why is this better than using `getRandomInternal().nextGaussian()` ?

Reply:
I think, this class is just another level of abstraction where all the types of generator functions are gathered. Why don't we use `getRandomInternal().nextInt()` instead of `nextInt()` method? :)

Reply:
Because `getRandomInternal().nextInt()` is new, and `nextInt()` was already there for several years, so removing it would have been hard. 

`getRandomInternal()` was introduced so it can be used to expose things which aren't currently exposed through Faker, such as, say, `nextGaussian()` ;-)

Reply:
no idea why, may be need to go to git history to answer that question, however what is the benefit here or what kind task will it solve ?
However based on code methods from this class are usually used internally while `faker.number` is more like public..
Since `RandomService` is public the rule is likely not 100% satisfied

Reply:
I see, that from your point of view guys, these changes don't make much sense for now, will close PR then, until it might be required.